src/job.rs

```diff
@@ -897,13 +911,51 @@ pub fn job_send_msg(context: &Context, msg_id: MsgId) -> Result<()> {
     Ok(())
 }
 
+fn add_imap_deletion_jobs(context: &Context) -> sql::Result<()> {
+    if let Some(delete_server_after) = context.get_config_delete_server_after() {
+        let threshold_timestamp = time() - delete_server_after;
+
+        // Select all expired messages which don't have a
+        // corresponding message deletion job yet.
+        let msg_ids = context.sql.query_map(
+            "SELECT id FROM msgs
```

Comment:
Threading into bikeshedding territory here, but the PR isn't consistent in how it formats SQL statements, earlier it was using \-terminated lines.

Reply:
Fixed in a separate commit.